probe/ebpf/flow.c

```diff
@@ -199,32 +199,34 @@ static inline void fill_ipv6(struct __sk_buff *skb, int offset, __u8 *dst, __u64
 	update_hash_word(hash, w);
```

Comment:
edit the line 111  : 	layer->_hash = FNV_BASIS ^ hash_src ^ hash_dst **^ protocol**